backend/serve.go

```diff
@@ -85,11 +85,11 @@ func defaultGRPCMiddlewares(opts ServeOpts) []grpc.ServerOption {
 	grpcMiddlewares := []grpc.ServerOption{
 		grpc.MaxRecvMsgSize(opts.GRPCSettings.MaxReceiveMsgSize),
 		grpc.StreamInterceptor(grpc_middleware.ChainStreamServer(
-			otelgrpc.StreamServerInterceptor(),
+			otelgrpc.StreamServerInterceptor(), //nolint:staticcheck
```

Comment:
not sure what to do here. The library suggest to use [NewServerHandler](https://pkg.go.dev/go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc#NewServerHandler) but that does not fit here. 

dependabot/go_modules/go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc-0.46.0

Reply:
Taking a look :eyes:

Reply:
@andresmgot I pushed a fix to this PR's branch, please take a look: https://github.com/grafana/grafana-plugin-sdk-go/pull/804/commits/1ed8e7be74e4cd750de1de6d392bb60b4f648f28

I have also done the same for the draft PR in core: https://github.com/grafana/grafana/pull/78069/commits/19843d84d7390852879e71f17125ae1e40d641b7